entries/z/zerotier.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "ZeroTier": {
+    "domain": "zerotier.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://accounts.zerotier.com/realms/zerotier/account/#/account-security/signing-in",
+    "categories": [
+      "remote",
```

Comment:
Does ZeroTier provide a remote access platform like GoTo, AnyDesk, ngrok, or TeamViewer?

Reply:
Similar use case to ngrok where you can allow a local network resource available remotely, though I guess that would be through a software-defined WAN or VPN style setup and not a simple reverse proxy. After comparison with GoTo/AnyDesk/TeamViewer I don't think the remote category fits.